open_academy/models/session.py

```diff
@@ -1,14 +1,47 @@
-from odoo import models, fields
+from datetime import date
+
+from odoo import models, fields, api, _
+
+from odoo.exceptions import ValidationError
 
 
 class Session(models.Model):
     _name = "session"
     _description = "Session"
 
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=date.today())
     duration = fields.Float()
     seats = fields.Integer(string="Number of seats")
+    taken_seats = fields.Integer()
+    percent_seats = fields.Integer(string="Percentage of taken seats", compute="_compute_seats")
+    active = fields.Boolean(default=True)
     instructor_id = fields.Many2one("res.partner", domain="[('instructor', '=', True)]")
     course_id = fields.Many2one("course")
     attendee_ids = fields.Many2many("res.partner")
+
+    @api.depends("seats", "taken_seats")
+    def _compute_seats(self):
+        for record in self:
+            record.percent_seats = 0
```

Comment:
This operation does not make sense, you are assigning a 0 and then the value of the operation on line 29.

Remove it